### ISSUE TITLE
Streamer Offline

### ISSUE THREAD

**Issue by FadingToJosh** [pyroch/TwitchGrobs#6]
Per your message, I'm opening an issue.
![chrome_qDCrXHPJNV](https://user-images.githubusercontent.com/1158510/113609554-c9847680-9600-11eb-8b8f-7fee0b168f6f.png)

**Comment by pyroch** [comment]
Hmm... Just tested - everything working for me. Was it just a single case? If so it's okay then not much to worry about.

**Comment by battlekicker** [comment]
Have the same problem. Always show this messagebox. Even if I press the button, it appears again after a while.

**Comment by skupfer** [comment]
Reporting the same. Even if offline it should continue but it gets stuck as soon as this message comes up (as in if you don't press Ok you are going to have a bad time gaining drops). If I am asleep I have to be sure this program is running properly.

Today we got new drops. I added all streamers to the list. Currently it is watching frost_ and after 15 or so minutes this message came up and I would like to know why. It states watching now: frost_, the streamer is online and streaming. No way something could have triggered this error. After clicking Ok it switched to another streamer...
